tests/dummy/app/templates/application.hbs

```diff
@@ -201,6 +201,20 @@
 </section>
 
 <section class="component-doc">
+  <h2 class="mdc-typography--title">Progress Bar</h2>
+
+  <h3 class="mdc-typography--subheading2">Linear Progress Determinate</h3>
+  {{mdc-linear-progress progress=myProgress}}
```

Comment:
It might be nice to have an ember input bound to `{{myProgress}}` for the demo so you can see it change.